src/data/platforms.yml

```diff
@@ -95,3 +95,7 @@
   type: language
   name: Electron
   case_style: camelCase
+- slug: capacitor
```

Comment:
@PeloWriter Should this file be deleted?
I believe it's outdated, could you please confirm?
Otherwise we need to fill out some other platforms missing.

Reply:
It's not needed.